CMakeLists.txt

```diff
@@ -1,13 +1,13 @@
 
 #===============================================================================
-#      The main build file for building EAL using CMake.
+#      The main build file for building ale using CMake.
 #===============================================================================
 # CMake initialization
 
 # Specify the required version of CMake.
 # cmake 3.10 required for ctest/gtest integration
 cmake_minimum_required(VERSION 3.10)
-project(eal VERSION 0.0.1 DESCRIPTION "Ephemeris Abstraction Library")
+project(ale VERSION 0.0.1 DESCRIPTION "Ephemeris Abstraction Library")
```

Comment:
Below the name was changed to Abstraction Library for Ephemerides I think - we should be consistent.